infra/ci/beta_patch

```diff
@@ -2,11 +2,12 @@
 
 set -ev
 
-export BETA_RELEASE=false  # disable this flag until after commit so Travis will not pickup this push
-CURRENT_WORKGING_BRANCH=$TRAVIS_BRANCH
+#export BETA_RELEASE=false  # disable this flag until after commit so Travis will not pickup this push
+CURRENT_WORKGING_BRANCH=$BETA_RELEASE_BRANCH
```

Comment:
Spelling mistake in variable

Reply:
LOL, nice catch. It worked so far because I was using my IDE to reference that variable :)